### DISCUSSION TITLE
Heap Trace with static linking of the ONNX libraries

### DISCUSSION THREAD

**Discussion by ArtCoreIst** [microsoft/onnxruntime#12960] [Other Q&A]
Hi all,

For some reason I'm using the ONNX Runtime via static linking the ONNX libraries. When in the debug mode (using the ONNX libraries build with the Debug config option), there happens a long "heap trace" session before the main returns with exit status 0. I want to disable this heap tracing operation but can't find any explanation on the topic. 

I would be grateful if anyone can help me with this issue.

Thanks a lot in advance.

**Comment by pranavsharma** [comment]
Can you provide an example of this heap trace looks like and steps to repro?
